docs/modules/getting-started/pages/security.adoc

```diff
@@ -0,0 +1,337 @@
+        this.factoryId = 1;
+        this.classId = 1;
+    }
+    readPortable(reader) {
+        this.username = reader.readString('username');
+        this.endpoint = reader.readString('password');
+        this.password = reader.readString('endpoint');
+    }
+    writePortable(writer) {
+        writer.writeString('username', this.username);
+        writer.writeString('password', this.password);
+        writer.writeString('endpoint', this.endpoint);
+    }
+}
+function usernamePasswordCredentialsFactory(classId) {
+    if (classId === 1) {
+        return new UsernamePasswordCredentials();
+    }
+    return null;
+}
+(async () => {
+    try {
+        const client = await Client.newHazelcastClient({
+            clusterName: 'hello-world',
+            serialization: {
+                portableFactories: {
+                    1: usernamePasswordCredentialsFactory
+                }
+            },
+            customCredentials: new UsernamePasswordCredentials('member1', 's3crEt', '192.168.1.101:5701')
+        });
+        console.log(await client.getDistributedObjects());
+    } catch (error) {
+        console.error('Error occurred:', error);
+    }
+})();
+----
+. Run this script in the IDE.
+
+====
+
+When looking at the client terminal, you can see that the member side has authenticated and accepted the client connection.
+
+[source,shell]
+----
+INFO:hazelcast.lifecycle:HazelcastClient 5.1 is STARTING
+INFO:hazelcast.lifecycle:HazelcastClient 5.1 is STARTED
+INFO:hazelcast.connection:Trying to connect to Address(host=127.0.0.1, port=5701)
+INFO:hazelcast.lifecycle:HazelcastClient 5.1 is CONNECTED
+INFO:hazelcast.connection:Authenticated with server Address(host=192.168.1.9, port=5701):63b2a2ce-85f6-413f-8ce9-6058a748e4b9, server version: 5.0, local address: Address(host=127.0.0.1, port=36006)
+INFO:hazelcast.cluster:
+
+Members [1] {
+	Member 192.168.1.101:5701 - 63b2a2ce-85f6-413f-8ce9-6058a748e4b9
+}
+
+INFO:hazelcast.client:Client started
+----
+
+If you try to connect a client without any credentials or with credentials different than the configured ones on the member side,
+you can see the connection is refused by the member.
+
+[source,shell]
+----
+INFO:hazelcast.lifecycle:HazelcastClient 5.1 is STARTING
+INFO:hazelcast.lifecycle:HazelcastClient 5.1 is STARTED
+INFO:hazelcast.connection:Trying to connect to Address(host=127.0.0.1, port=5701)
+INFO:hazelcast.connection:Connection(id=0, live=False, remote_address=None) closed. Reason: Failed to authenticate connection
+WARNING:hazelcast.connection:Error during initial connection to Address(host=127.0.0.1, port=5701)
+----
+
+== Step 3. Shut Down the Cluster
+
+Shut down the cluster you've created in this tutorial so that you can start a fresh one when you
+move to the other tutorials. To shutdown, close the terminals in which the members are running or press kbd:[Ctrl+C] in each terminal.
+
+== Next Steps
+
+See xref:security:overview.adoc[Securing a Cluster] if you're
```

Comment:
```suggestion
If you're interested in learning more about the topics introduced in this tutorial, see xref:security:overview.adoc[] 
```

Put the information that users want to know first before telling them where to go.